java/src/main/java/org/wildfly/openssl/SSL.java

```diff
@@ -88,8 +91,12 @@ static void init() {
                             }
                         }
                     } else {
-                        Runtime.getRuntime().load(libPath);
-                        instance = new SSLImpl();
+                        try {
+                            Runtime.getRuntime().load(libPath);
+                            instance = new SSLImpl();
+                        } catch (Throwable t) {
+                            throw new RuntimeException("Failed to load wfssl native library from specified path: " + libPath, t);
```

Comment:
If we are throwing this exception is there not a more appropriate type we can use?